pgaudit.c

```diff
@@ -1100,6 +1117,12 @@ log_select_dml(Oid auditOid, List *rangeTabls)
             case RELKIND_MATVIEW:
                 auditEventStack->auditEvent.objectType = OBJECT_TYPE_MATVIEW;
                 break;
+            case RELKIND_PARTITIONED_TABLE:
```

Comment:
Please space these case statements to match the others.

Reply:
ok, will modify it